default.nix

```diff
@@ -85,6 +85,76 @@ let
       });
     };
   });
+  rawDockerImage = let
+    topologyFile = pkgs.writeText "topology.yaml" ''
+      wallet:
+        relays:
+          [
+            [
+              { host: nodes.awstest.iohkdev.io }
+            ]
+          ]
+        valency: 1
+        fallbacks: 7
```

Comment:
We should update the topology to one closer to what we really want. Note that since #1668 the `valency` and `fallbacks` fields are not used and this info is encoded directly in the lists of hosts.

For exchanges, we want them to subscribe to 3 relays, with other relays as backups. We can either use the primary relays, or we can decide to do something else.

Reply:
@dcoutts could we tell them to use public relays, but subscribe to 3?

Reply:
@dcoutts https://issues.serokell.io/issue/DEVOPS-462
a new PR with the mainnet relays and better fallbacks will follow in the future
the current topology is only for testnet